### DISCUSSION TITLE
Dax Studio compatible with Windows 11

### DISCUSSION THREAD

**Discussion by Evert62** [DaxStudio/DaxStudio#1310] [General]
Hello, can somebody tell me which version of Dax Studio is compatible with Windows 11?
Thanks for your reaction!
Greetings, Evert

**Comment by dgosbell** [comment]
The current version should work fine with Windows 11. Is there something on https://daxstudio.org that made you think it was not supported? If so, can you share a link as I'd like to fix anything like that.
